src/index.html

```diff
@@ -102,6 +102,13 @@ <h2 class="govuk-heading-l">How it works</h2>
         <a href="/alerts/how-alerts-work" class="govuk-link govuk-link--no-visited-state govuk-!-font-weight-bold">How emergency alerts work</a>
       </p>
     </div>
+    <div class="govuk-grid-column-one-half">
+      <div class="responsive-embed responsive-embed--16by9">
+        <div class="responsive-embed__wrapper">
```

Comment:
I am probably missing something, but why is `<div class="responsive-embed__wrapper">` class div inside `<div class="responsive-embed responsive-embed--16by9">`? Shouldn't the wrapper be the outside one?

Reply:
This is an example of the [block-element-modifier (BEM)](https://css-tricks.com/bem-101/) methodology for writing CSS:

- The block `responsive-embed` is the top-level thing
- The element `responsive-embed__wrapper` is a child of the block (denoted by two underscores)
- The modifier `responsive-embed--16by9` alters the block in some way (denoted by two dashes)

Maybe `wrapper` isn’t the best name, but it sort of makes sense if you think of it as wrapping the `<iframe>` itself. What do you think?

Reply:
I like `__outer` – changed in e1f0e13

Reply:
I like it, too, sorry for late reply 👍🏼